src/foundry/foundry.js/pixi/containers/placeableObject.d.ts

```diff
@@ -1,425 +1,418 @@
 
-  /**
-   * Activate interactivity for the Placeable Object
-   */
-  activateListeners(): void;
+    /**
+     * Callback actions which occur on a double right-click event to configure properties of the object
+     * @see MouseInteractionManager#_handleClickRight2
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onClickRight2(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Create a standard MouseInteractionManager for the PlaceableObject
-   */
-  protected _createInteractionManager(): NonNullable<this['mouseInteractionManager']>;
+    /**
+     * Callback actions which occur when a mouse-drag action is first begun.
+     * @see MouseInteractionManager#_handleDragStart
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftStart(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Actions that should be taken for this Placeable Object when a mouseover event occurs
-   * @param options - (default: `{}`)
-   * @param hoverOutOthers - (default: `true`)
-   */
-  protected _onHoverIn(event: PIXI.InteractionEvent, options?: { hoverOutOthers: boolean }): boolean | void;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragMove
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftMove(event: PIXI.InteractionEvent): void;
 
-  /**
-   * Actions that should be taken for this Placeable Object when a mouseout event occurs
-   */
-  protected _onHoverOut(event: PIXI.InteractionEvent): boolean | void;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragDrop
+     * @param event - The triggering canvas interaction event
+     */
+    protected _onDragLeftDrop(event: PIXI.InteractionEvent): Promise<false | this | D[]>;
 
-  /**
-   * Callback actions which occur on a single left-click event to assume control of the object
-   */
-  protected _onClickLeft(event: PIXI.InteractionEvent): boolean | null;
+    /**
+     * Callback actions which occur on a mouse-move operation.
+     * @see MouseInteractionManager#_handleDragCancel
+     * @param event - The triggering mouse click event
+     */
+    protected _onDragLeftCancel(event: MouseEvent): void;
+  }
 
-  /**
-   * Callback actions which occur on a double left-click event to activate
-   */
-  protected _onClickLeft2(event: PIXI.InteractionEvent): void;
+  namespace PlaceableObject {
+    interface ControlOptions {
+      /**
+       * Release any other controlled objects first
+       */
+      releaseOthers?: boolean;
+    }
+
+    interface ReleaseOptions {
+      trigger?: boolean;
+    }
+  }
+}
 
+interface Vision {
   /**
-   * Callback actions which occur on a single right-click event to configure properties of the object
+   * @remarks
+   * This is required but has ben set to optional because of pointsource
```

Comment:
Is this still necessary? In any case, there's a typo, 'ben' instead of 'been'?

Reply:
Yes, still necessary.